src/parser/sparqlParser/SparqlQleverVisitor.h

```diff
@@ -249,12 +249,42 @@ class SparqlQleverVisitor : public SparqlAutomaticVisitor {
 
   antlrcpp::Any visitOrderClause(
       SparqlAutomaticParser::OrderClauseContext* ctx) override {
-    return visitChildren(ctx);
+    vector<OrderKey> _orderBy;
```

Comment:
not a member, so no `_` in the name.
And it's rather `orderConditions` (the name reflects that there's multiple of them).